adoc/bp_sp_migration.adoc

```diff
@@ -82,6 +82,9 @@ For more information, see <<sp.migration.postgresql>>.
 Comprehensive documentation for executing service pack migration scenarios is located in the {sls} documentation chapter https://www.suse.com/documentation/sles-12/book_sle_deployment/data/cha_update_sle.html.
 
 
+////
+This is a repeat of the information in bp.postgresql.database.migration.
+Removing for now, so we're not maintaining two copies. -LKB
```

Comment:
ok, but we are still referencing the `[[sp.migration.postgresql]]` ID/label from elsewhere.  Thus means we either must remove the dead pointers or keep this section for the moment.  Here are my grep results:

```grep sp.migration.postgresql adoc/*c
adoc/bp_sec_mgr_21_migration.adoc:For more information about the PostgreSQL upgrade, see <<sp.migration.postgresql>>.
adoc/bp_sp_migration.adoc:For more information, see <<sp.migration.postgresql>>.
adoc/bp_sp_migration.adoc:[[sp.migration.postgresql]]
adoc/bp_sp_migration.adoc:For more information, see <<sp.migration.postgresql>>.
adoc/bp_sp_migration.adoc:For more information, see <<sp.migration.postgresql>>.```